docs/user-manual.md

```diff
@@ -1,229 +1,263 @@
 
-$ kubectl create secret generic github-app-secret \
-    -n meows \
+$ kubectl create secret generic github-cred -n meows \
     --from-literal=app-id=${GITHUB_APP_ID} \
     --from-literal=app-installation-id=${GITHUB_APP_INSTALLATION_ID} \
     --from-file=app-private-key=${GITHUB_APP_PRIVATE_KEY_PATH}
 ```
 
-In addition to this, the admission webhook requires a TLS certificate.
-You should use [`github.com/jetstack/cert-manager`](https://github.com/jetstack/cert-manager)
-or create a certificate by yourself.
+If you want to use a Personal Access Token (PAT), create a PAT following [the official documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
+Then you need to set the `repo` scope to the token.
 
-This document does not give you a comprehensive list of what you should deploy.
-Please refer to the manifests under `config/` for further information.
+And create a secret as follows:
 
-How to create Slack App
------------------------
+```bash
+$ GITHUB_TOKEN=<your PAT>
 
-Slack agent notifies users whether CI succeeds or not and receives messages to
-extend a `Pod` lifetime.
-So, users have to prepare a Slack App to send messages to and run a WebSocket client
-to watch button events.
+$ kubectl create secret generic github-cred -n meows \
+    --from-literal=token=${GITHUB_TOKEN}
+```
 
-Here's a procedure for how to configure the Slack App.
+### Deploying Controller
 
-1. Go to [this](https://api.slack.com/apps) page.
-1. Click the **Create New App** button.
-   - Choose **From scratch**.
-   - Fill the application name field and choose a Slack workspace.
-1. Go to **Socket Mode** from the sidebar.
-   - Enable **Enable Socket Mode**.
-   - Create App-Level Token on the windows coming up and keep the generated App Token.
-1. Go to **OAuth & Permissions** from the sidebar.
-   - Add the `chat:write` permission under **Bot Token Scopes**.
-1. Go to **Basic Information** from the sidebar.
-   - Make sure **Bots** is enabled in `Add features and functionality`
-   - Click **Install(Reinstall) to Workspace** in `Install your app` and (re)install the bot in your desired channel.
-1. Go to **OAuth & Permissions** from the sidebar again.
-   - Keep **Bot User OAuth Token**.
-1. Open your Slack desktop app and go to your desired channel.
-   - Click the `i` button on the top right corner.
-   - Click **more** and then **Add apps**.
-   - Add the created Slack App to the channel.
+Currently, meows can manage one organization.
+So set the target organization by `meows-cm` ConfigMap.
+
+```bash
+$ GITHUB_ORG=<your Organization>
+$ kubectl create configmap meows-cm -n meows \
+    --from-literal=organization=${GITHUB_ORG}
+```
+
+After that deploy the controller.
+
+```bash
+$ MEOWS_VERSION=v0.5.0
+$ kubectl apply -k github.com/cybozu-go/meows/config/controller?ref=${MEOWS_VERSION}
+```
 
-How to deploy Slack agent
--------------------------
+### Deploying Slack Agent
```

Comment:
Is Slack Agent optional?
If so, could you write that it will be needed to notify CI completion and to extend Runner?

Reply:
It's not optional now.
Currently, the runner pod's default extension time is 20 min.
So without the Slack agent, the failed pod will remain for 20 minutes in vain.